src/types/GameType.ts

```diff
@@ -0,0 +1,18 @@
+export interface GameItem extends GameInputType {
+  id: string;
+}
+
+export interface GameInputType {
+  name: string;
+  genres: string[];
+  minPlayer: number;
+  description: string;
+  maxPlayer: number;
+  playDuration: number;
+  dateAdded?: string;
+  available: boolean;
+  pictures: string[];
+  size: String;
+  minAge: number;
+  condition: string;
+} 
```

Comment:
Naming it `GameInputStyle` is a bit weird. Maybe have everything in `GameItem`, but here in the type for input, we use `pick` (or something, not sure on the TS syntax) to choose the ones we actually use for user changes?